app/javascript/lib/validation/validate-file-content.js

```diff
@@ -425,6 +425,9 @@ async function parseFile(file, fileType, fileOptions={}, sizeProps={}) {
     // get any unhandled or deliberate short-circuits
     if (error instanceof ParseException) {
       parseResult.issues.push(['error', error.key, error.message])
+    } else if (error instanceof TypeError) {
+      const msg = 'File cannot be uploaded in it\'s current state. Please reach out for assistance'
```

Comment:
I think this is great that we're catching/handling these now.  Non-blocking: If possible, it would be amazing if we could get more info on the file rather than "cannot be uploaded", like an offending line number or character string.

Reply:
Is it accurate to say that "non UTF-8" (or maybe non-standard?) characters were detected in the file? 

It might be worth making the error message specific enough that we'll remember what the root cause of the issue is. (Because it's really hard to see/find the hidden characters).

Reply:
Yup, I like the idea of a `File cannot be parsed: non-UTF-8 characters are present.` message

Reply:
It seems incorrect to tell the user this failed because non-UTF-8 characters were in the file, because the three characters in the test case -- [п](https://util.unicode.org/UnicodeJsps/character.jsp?a=041F) and [»](https://util.unicode.org/UnicodeJsps/character.jsp?a=00BB) and [ї](https://util.unicode.org/UnicodeJsps/character.jsp?a=0457) -- are all valid UTF-8.  

If we essentially keep the current generic message, let's use "[its](https://www.merriam-webster.com/words-at-play/when-to-use-its-vs-its)" and end the sentence with a period.

To Jon's comment, to my understanding we won't know line number, but I'd be curious if we'd have any more specific error information (whether offending string as he wonders, or anything else).

Reply:
unfortunately yeah we won't have the line number since it's occurring within io.js and all that has is chunk start and chunk end which don't seem super helpful to tell the user. 

I also kept this message more generic because the error type of "TypeError" could be caused by other things than just hidden/special characters and the error message that comes with this error is:  "Failed to execute 'decode' on 'TextDecoder': The encoded data was not valid.". So it's just about an decodable file so I didn't want to make the message incorrectly specific.